metaflow/plugins/kfp/kfp_cli.py

```diff
@@ -451,6 +452,37 @@ def _convert_value(param: parameters.Parameter):
     "If not set, METAFLOW_NOTIFY_ON_SUCCESS is used from Metaflow config or environment variable",
     show_default=True,
 )
+@click.option(
+    "--recurring-run-enable/--no-recurring-run-enable",
+    "recurring_run_enable",
+    default=False,
```

Comment:
For customers defaulting to `True` make more sense.

Reply:
It'd be nice to hear more why, shouldn't recurring runs be disabled by default given that they could use up resources and make it an explicit parameter/step for the user to take to enable recurring runs?

Reply:
I'm actually a bit confused why a recurring run is always created. Could you please explain why a suspended recurring run is chosen over not adding recurring run?

Reply:
https://docs.google.com/document/d/1cR-R3mqrAugM5veiceIbrwllhupTmLS8jBgBJOIuz0w/edit?pli=1#heading=h.x8ue7bngf6ku

thanks!